risedev.yml

```diff
@@ -38,10 +38,10 @@ risedev:
 
     # If you want to create source from Kafka, uncomment the following lines
     # Note that kafka depends on zookeeper, so zookeeper must be started beforehand.
-    # - use: zookeeper
-    #   persist-data: true
-    # - use: kafka
-    #   persist-data: true
+    - use: zookeeper
+      persist-data: false
+    - use: kafka
+      persist-data: false
```

Comment:
Please don't change the default config if not necessary.

Reply:
Sorry, I commit it by mistake.